### DISCUSSION TITLE
Noob: How to do first steps - models?

### DISCUSSION THREAD

**Discussion by Knubbel** [Mikubill/sd-webui-controlnet#669] [General]
Hi,
I am a total noob here, so please forgive my lack of understanding. On Windows 10 64bit.

1)
First issue I stumble upon when following the instructions is how to install a model and which one.
Instructions say "inside the models/ControlNet folder.", but there is not /ControlNet folder anywhere.
I do see both a `C:\Users\[username]\stable-diffusion-webui\models\` folder and one `C:\Users\[username]\stable-diffusion-webui\extensions\sd-webui-controlnet\models`
In which one should I create a subfolder called "ControlNet" and put the model file?

2)
I am a little overwhelmed by the options on models. Basically I just want to have the extension impact the pose of human models in images. So I do wonder which one I should pick as a beginner.
Pretrained? Pre-extracted? Which one?
I read a suggestion that I do use `Control_sd15_Scibble` Since the pth-file is 5.71 GB large and there is a safetensors-file just 0,7 GB large I wonder what the advantages are to either.

**Comment by 5shekel** [comment]
> I do see both a C:\Users\[username]\stable-diffusion-webui\models\ folder and one C:\Users\[username]\stable-diffusion-webui\extensions\sd-webui-controlnet\models

goto `extensions\sd-webui-controlnet\models`  and git the models (this might take some time. 
this will create a  `ControlNet-v1-1` subfolder under models folder. it will search anyhting under that path incllduing subfolders.

huggin recommends.
```bash
cd C:\Users\[username]\stable-diffusion-webui\extensions\sd-webui-controlnet\models
git lfs install
git clone https://huggingface.co/lllyasviel/ControlNet-v1-1
```
